Screenbox/Package.appxmanifest

```diff
@@ -255,88 +437,110 @@
           </uap:FileTypeAssociation>
         </uap:Extension>
         <uap:Extension Category="windows.fileTypeAssociation">
-          <uap:FileTypeAssociation Name="mpeg-ts">
+          <uap:FileTypeAssociation Name="wma">
+            <uap:DisplayName>ms-resource:ManifestResources/FileDisplayNameWindowsMediaAudio</uap:DisplayName>
+            <uap:EditFlags OpenIsSafe="true"/>
+            <uap:Logo>Assets\Icons\AudioFileAssociation.png</uap:Logo>
             <uap:SupportedFileTypes>
-              <uap:FileType ContentType="video/mp2t">.ts</uap:FileType>
-              <uap:FileType ContentType="video/mp2t">.m2t</uap:FileType>
-              <uap:FileType ContentType="video/mp2t">.m2ts</uap:FileType>
-              <uap:FileType ContentType="video/mp2t">.mts</uap:FileType>
+              <uap:FileType ContentType="audio/x-ms-wma">.wma</uap:FileType>
             </uap:SupportedFileTypes>
-            <uap:EditFlags OpenIsSafe="true"/>
-            <uap:DisplayName>ms-resource:ManifestResources/FileDisplayNameMPEGTS</uap:DisplayName>
-            <uap:Logo>Assets\Icons\FileAssociation.png</uap:Logo>
           </uap:FileTypeAssociation>
         </uap:Extension>
         <uap:Extension Category="windows.fileTypeAssociation">
-          <uap:FileTypeAssociation Name="wavpack">
+          <uap:FileTypeAssociation Name="wmv">
+            <uap:DisplayName>ms-resource:ManifestResources/FileDisplayNameWindowsMediaVideo</uap:DisplayName>
+            <uap:EditFlags OpenIsSafe="true"/>
+            <uap:Logo>Assets\Icons\VideoFileAssociation.png</uap:Logo>
             <uap:SupportedFileTypes>
-              <uap:FileType ContentType="audio/x-wavpack">.wv</uap:FileType>
-              <uap:FileType ContentType="audio/x-wavpack-correction">.wvc</uap:FileType>
+              <uap:FileType ContentType="video/x-ms-wmv">.wmv</uap:FileType>
             </uap:SupportedFileTypes>
-            <uap:DisplayName>ms-resource:ManifestResources/FileDisplayNameWavPack</uap:DisplayName>
-            <uap:EditFlags OpenIsSafe="true"/>
-            <uap:Logo>Assets\Icons\FileAssociation.png</uap:Logo>
           </uap:FileTypeAssociation>
         </uap:Extension>
-        <uap:Extension Category="windows.fileTypeAssociation">
-          <uap:FileTypeAssociation Name="aiff">
+        <!--<uap:Extension Category="windows.fileTypeAssociation">
+          <uap:FileTypeAssociation Name="wpl">
+            <uap:DisplayName>Windows Media Player Playlist</uap:DisplayName>
+            <uap:EditFlags OpenIsSafe="true"/>
+            <uap:Logo>Assets\Icons\PlaylistFileAssociation.png</uap:Logo>
             <uap:SupportedFileTypes>
-              <uap:FileType ContentType="audio/aiff">.aiff</uap:FileType>
-              <uap:FileType ContentType="audio/aiff">.aif</uap:FileType>
-              <uap:FileType ContentType="audio/aiff">.aifc</uap:FileType>
+              <uap:FileType ContentType="application/vnd.ms-wpl">.wpl</uap:FileType>
```

Comment:
Same here. Why did this file type association get commented out? What's the plan for these file types?